.github/workflows/fbgemmci.yml

```diff
@@ -248,6 +248,67 @@ jobs:
         python -c "import fbgemm_gpu"
         python -c "import fbgemm_gpu.split_embedding_codegen_lookup_invokers"
 
+  build_amd_gpu:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - name: Free space
+      run: sudo rm -rf /usr/local/android /usr/share/dotnet /usr/local/share/boost /opt/ghc /usr/local/share/chrom* /usr/share/swift /usr/local/julia* /usr/local/lib/android
+
+    - uses: actions/checkout@v2
+
+    - name: Install ROCm
```

Comment:
nit: could you name it ROCm 5.1.1 (or a certain ROCm version used in this CI)? We want to use the same name as that of CUDA.

Reply:
> 

Sure, I'll add the version in the next commit. Thank you for reminding me.